src/jaxoplanet/experimental/starry/light_curves.py

```diff
@@ -7,6 +7,10 @@
 import scipy
 
 from jaxoplanet.experimental.starry.basis import A1, A2_inv, U
+from jaxoplanet.experimental.starry.multiprecision import (
+    basis as basis_mp,
+    utils as utils_mp,
+)
```

Comment:
I think mpmath should stay an optional dependency so this import should probably be conditional. How about moving this import into the `surface_light_curve` function and then only try and do the import when `higher_precision=True`? Then we could also raise a better error message (e.g. "higher_precision requires mpmath...").

Reply:
I completely forgot to fix that. Thanks!